stories/cloud_cryptomining.yml

```diff
@@ -13,7 +13,7 @@ detections:
   - detection_id: c6ddbf53-9715-49f3-bb4c-fb2e8a309cda
     name: Cloud Compute Instance Created With Previously Unseen Instance Type
     type: splunk
-  - detection_id: 8dbcf362-ae43-44ea-9084-77db4bed2c68
+  - detection_id: bc24922d-987c-4645-b288-f8c73ec194c4
```

Comment:
not sure why this id changed. Also, add your MLTK detection object in this story

Reply:
I changed it b/c the detection_id referenced doesn't exist. The one I changed it to is the correct one accd to my searches and a brief chat with @trogdorsey  last week.

Reply:
right on! are we adding your MLTK detection to this story?

Reply:
Yes. I made the commit just now.